Readme.md

```diff
@@ -43,14 +43,17 @@ import { bins, net } from "capi"
 
 const bin = bins({ polkadot: ["polkadot", "v0.9.38"] })
 
-// The Polkadot relay chain
-export const polkadot = net.ws({ url: "wss://rpc.polkadot.io/" })
-
 // A Polkadot development network
 export const polkadotDev = net.dev({
   bin: bin.polkadot,
   chain: "polkadot-dev",
 })
+
+// The Polkadot relay chain
+export const polkadot = net.ws({
+  url: "wss://rpc.polkadot.io/",
+  targets: { dev: polkadotDev },
```

Comment:
should we do this in the capi `nets.ts` file?

Reply:
No, as it would require separate server instances for differently-targeted examples.